tests/devapps/LinuxApp/TestApp.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Do we need a separate dev app for this? Would [NetCoreTestApp](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/tree/master/tests/devapps/NetCoreTestApp) work?

Reply:
Yes, the existing netcore apps would work

Reply:
I tried. It has references to other project and require compilation. The LinuxApp can be used with NuGet and be exclusive for Linux testing.

Reply:
All our dev apps use project references. It can be misleading to have dev apps use nuget references. I'd much rather fix the build so that it completes on Linux. This is the right thing to do.

But to get this fixed quickly how about this? Change your manual test instructions to use this sample project https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/tree/master/1-Calling-MSGraph/1-1-AzureAD/Console-Interactive-MultiTarget - this is a .net core project and uses AcquireTokenInteractive

Reply:
Removed the TestApp